changelog/v0.35.0/bump-k8s-129.yaml

```diff
@@ -0,0 +1,54 @@
+changelog:
+- type: BREAKING_CHANGE
+  description: >
+    Bump dependencies to support Kubernetes 1.29
+  issueLink: https://github.com/solo-io/gloo/issues/8978
+  resolvesIssue: false
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: api
+  dependencyTag: v0.29.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: apiextensions-apiserver
+  dependencyTag: v0.29.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: apimachinery
+  dependencyTag: v0.29.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: client-go
+  dependencyTag: v0.29.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: sigs.k8s.io
+  dependencyRepo: code-generator
+  dependencyTag: v0.29.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: envoyproxy
+  dependencyRepo: go-control-plane
+  dependencyTag: v0.11.1
+- type: DEPENDENCY_BUMP
+  dependencyOwner: envoyproxy
+  dependencyRepo: protoc-gen-validate
+  dependencyTag: v1.0.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: onsi
+  dependencyRepo: ginkgo/v2
+  dependencyTag: v2.14.0
+- type: DEPENDENCY_BUMP
+  dependencyOwner: onsi
+  dependencyRepo: gomega
+  dependencyTag: v1.30.0
+- type: DEPENDENCY_BUMP
+  dependencyOwner: go.uber.org
+  dependencyRepo: zap
+  dependencyTag: v1.26.0
+- type: DEPENDENCY_BUMP
+  dependencyOwner: solo-io
+  dependencyRepo: k8s-utils
+  dependencyTag: v0.7.2
+- type: DEPENDENCY_BUMP
+  dependencyOwner: solo-io
+  dependencyRepo: cloud-builders
+  dependencyTag: v0.9.1
```

Comment:
Last time we updated we also updated `go-utils` (https://github.com/solo-io/solo-kit/blob/e7296cef16898146ed5eedba199c54df2f47ede0/changelog/v0.34.0/bump-k8s-128.yaml#L39) which looks like its still on 1.28. Do we need to bump that as well?

Reply:
go-utils does not import any k8s libs so there's no need to update it. The last bump only updated some other libraries used by k8s but it isn't necessary to update them